recipe/meta.yaml

```diff
@@ -19,16 +19,18 @@ requirements:
     - pip
     - python
   run:
+    - python
     - fluids >=0.1.68
     - pandas
-    - python
     - scipy
-    - pint
-    - sympy
     - coolprop
-    - matplotlib
```

Comment:
For me, it looks fine. `thermo` has some plotting features, but maybe it should be delegated to the user instead the lib.

Reply:
@CalebBell what do you think about moving `matplotlib` to the test's dependency?

Reply:
All checks passed, so it's ready to merge